src/databricks/labs/ucx/source_code/known.json

```diff
@@ -22808,6 +22808,363 @@
+    "theano.tensor.nnet.nnet": [],
+    "theano.tensor.nnet.opt": [],
+    "theano.tensor.nnet.sigm": [],
+    "theano.tensor.nnet.tests": [],
+    "theano.tensor.nnet.tests.speed_test_conv": [],
+    "theano.tensor.nnet.tests.test_abstract_conv": [],
+    "theano.tensor.nnet.tests.test_blocksparse": [],
+    "theano.tensor.nnet.tests.test_bn": [],
+    "theano.tensor.nnet.tests.test_conv": [],
+    "theano.tensor.nnet.tests.test_conv3d2d": [],
+    "theano.tensor.nnet.tests.test_corr": [],
+    "theano.tensor.nnet.tests.test_corr3d": [],
+    "theano.tensor.nnet.tests.test_ctc": [],
+    "theano.tensor.nnet.tests.test_neighbours": [],
+    "theano.tensor.nnet.tests.test_nnet": [],
+    "theano.tensor.nnet.tests.test_opt": [],
+    "theano.tensor.nnet.tests.test_sigm": [],
+    "theano.tensor.opt": [],
+    "theano.tensor.opt_uncanonicalize": [],
+    "theano.tensor.raw_random": [],
+    "theano.tensor.shared_randomstreams": [],
+    "theano.tensor.sharedvar": [],
+    "theano.tensor.signal": [],
+    "theano.tensor.signal.conv": [],
+    "theano.tensor.signal.pool": [],
+    "theano.tensor.signal.tests": [],
+    "theano.tensor.signal.tests.test_conv": [],
+    "theano.tensor.signal.tests.test_pool": [],
+    "theano.tensor.slinalg": [],
+    "theano.tensor.sort": [],
+    "theano.tensor.subtensor": [],
+    "theano.tensor.tests": [],
+    "theano.tensor.tests._test_mpi_roundtrip": [],
+    "theano.tensor.tests.mlp_test": [],
+    "theano.tensor.tests.test_basic": [],
+    "theano.tensor.tests.test_blas": [],
+    "theano.tensor.tests.test_blas_c": [],
+    "theano.tensor.tests.test_blas_scipy": [],
+    "theano.tensor.tests.test_casting": [],
+    "theano.tensor.tests.test_complex": [],
+    "theano.tensor.tests.test_elemwise": [],
+    "theano.tensor.tests.test_extra_ops": [],
+    "theano.tensor.tests.test_fft": [],
+    "theano.tensor.tests.test_fourier": [],
+    "theano.tensor.tests.test_gc": [],
+    "theano.tensor.tests.test_inc_subtensor": [],
+    "theano.tensor.tests.test_io": [],
+    "theano.tensor.tests.test_keepdims": [],
+    "theano.tensor.tests.test_merge": [],
+    "theano.tensor.tests.test_misc": [],
+    "theano.tensor.tests.test_mpi": [],
+    "theano.tensor.tests.test_nlinalg": [],
+    "theano.tensor.tests.test_opt": [],
+    "theano.tensor.tests.test_opt_uncanonicalize": [],
+    "theano.tensor.tests.test_raw_random": [],
+    "theano.tensor.tests.test_shared_randomstreams": [],
+    "theano.tensor.tests.test_sharedvar": [],
+    "theano.tensor.tests.test_slinalg": [],
+    "theano.tensor.tests.test_sort": [],
+    "theano.tensor.tests.test_subtensor": [],
+    "theano.tensor.tests.test_type_other": [],
+    "theano.tensor.tests.test_utils": [],
+    "theano.tensor.tests.test_var": [],
+    "theano.tensor.tests.test_xlogx": [],
+    "theano.tensor.type": [],
+    "theano.tensor.type_other": [],
+    "theano.tensor.utils": [],
+    "theano.tensor.var": [],
+    "theano.tensor.xlogx": [],
+    "theano.tests": [],
```

Comment:
remove test packages, it's noise

Reply:
done

Reply:
<img width="551" alt="image" src="https://github.com/databrickslabs/ucx/assets/259697/eb3e9f2b-8b0a-4fb7-96de-f68b99592116">
they're in multiple places. please review diffs before opening PRs :)

Reply:
Done